markdown/cli.ts

```diff
@@ -164,8 +164,11 @@ program
             console.log(doc.metadata.slug);
           }
           const { body: h, attributes: metadata } = fm(doc.rawContent);
+          const concreteLocale =
+            options.locale === "all" ? "en-US" : options.locale;
```

Comment:
there should be `doc.locale` available. And this way it even works properly when building all locales.

Reply:
Thanks, it looks better now :)